app/contexts/NetworkContext.tsx

```diff
@@ -11,6 +12,18 @@ interface Network {
 
 const NetworkContext = createContext<Network>(undefined as any)
 
+export function networkMapper (network: EnvironmentNetwork): NetworkName {
```

Comment:
don't edit the context to export network config out, you are pulling network config out instead of reading into it.

Reverse the flow of this implementation, such that you expose this instead: `const { parseAddress } = useNetworkContext()`

Reply:
```suggestion
function networkMapper (network: EnvironmentNetwork): NetworkName {
```

don't export this